django/db/migrations/loader.py

```diff
@@ -321,3 +321,20 @@ def project_state(self, nodes=None, at_end=True):
         See graph.make_state for the meaning of "nodes" and "at_end"
         """
         return self.graph.make_state(nodes=nodes, at_end=at_end, real_apps=list(self.unmigrated_apps))
+
+
+def has_unapplied_migrations(connection):
```

Comment:
Largely similar to the code used in `showmigrations` management command, inspired by @timgraham's idea.